### ISSUE TITLE
Recipe Data Debug Mode

### ISSUE THREAD

**Issue by jayaddison** [openculinary/frontend#94]
**Is your feature request related to a problem? Please describe.**
We have data quality issues with some recipes; for example https://github.com/openculinary/frontend/issues/92

Currently it is challenging to debug the source of the problem.  There are multiple possible causes:

- Was the recipe/ingredient description incorrect on the original website?
- Did the [recipe-scrapers](https://github.com/hhursev/recipe-scrapers) software correctly extract the recipe data?
- Did RecipeRadar's [knowledge-graph](https://github.com/openculinary/knowledge-graph) correctly parse the ingredient names?
- Did the page render the recipe correctly?

**Describe the solution you'd like**
It should be possible to identify the source of data quality issues from inside the RecipeRadar user interface.

This ability should not be visible to users of the site unless they want to know about the details of the data.

A proposed solution is:

After the user clicks on a recipe image five times within five seconds, a new `Data` tab should appear.

![image](https://user-images.githubusercontent.com/55152140/79038649-df6c8080-7bd2-11ea-952d-29156aa1e929.png)

This should contain the following information:

* The URL(s) of the recipe
* The time the recipe was downloaded from the web
* The time the recipe was indexed into the RecipeRadar search engine
* The version of the RecipeRadar [crawler](https://github.com/openculinary/crawler) used when processing the recipe
* The current JSON recipe data using the latest RecipeRadar crawler

@queencitra would this be helpful?  Please provide any suggestions and improvements.

**Comment by jayaddison** [comment]
Currently the `crawler` JSON output can be retrieved using this command locally on a RecipeRadar server:

```sh
$ curl -s -XPOST -H 'Host: crawler' localhost:30080/crawl --data "url=https://tastykitchen.com/recipes/drinks/mango-pineapple-and-aloe-smoothie/" | json_pp 
{
   "image_src" : "https://tastykitchen.com/recipes/wp-content/uploads/sites/2/2014/06/Mango-Pineapple-and-Aloe-Smoothie-014a-410x318.jpg",
   "domain" : "tastykitchen.com",
   "time" : 5,
   "src" : "https://tastykitchen.com/recipes/drinks/mango-pineapple-and-aloe-smoothie/",
   "ingredients" : [
      {
         "description" : "8 ounces, fluid aloe vera drink, mangosteen and mango flavor",
         "index" : 0,
         "product" : {
            "category" : null,
            "plural" : "mangoes",
            "product" : "mango",
            "is_plural" : false,
            "contents" : [
               "mango"
            ],
            "ancestors" : [],
            "singular" : "mango",
            "product_parser" : "knowledge-graph"
         },
         "units" : "g",
         "quantity" : 226.8,
         "quantity_parser" : "ingreedypy+pint",
         "units_parser" : "ingreedypy+pint"
      },
      {
         "product" : {
            "contents" : [
               "mango"
            ],
            "singular" : "mango",
            "ancestors" : [],
            "product_parser" : "knowledge-graph",
            "is_plural" : false,
            "product" : "mango",
            "plural" : "mangoes",
            "category" : null
         },
         "index" : 1,
         "description" : "½ cups mango, frozen",
         "units_parser" : "ingreedypy+pint",
         "quantity_parser" : "ingreedypy+pint",
         "quantity" : 118.29,
         "units" : "ml"
      },
      {
         "product" : {
            "category" : null,
            "plural" : "pineapples",
            "product" : "pineapple",
            "is_plural" : false,
            "singular" : "pineapple",
            "ancestors" : [],
            "contents" : [
               "pineapple"
            ],
            "product_parser" : "knowledge-graph"
         },
         "index" : 2,
         "description" : "½ cups pineapple, frozen",
         "units_parser" : "ingreedypy+pint",
         "quantity" : 118.29,
         "quantity_parser" : "ingreedypy+pint",
         "units" : "ml"
      }
   ],
   "servings" : 1,
   "title" : "Mango, Pineapple and Aloe Smoothie",
   "rating" : 4,
   "directions" : [
      {
         "index" : 0,
         "description" : "Add all ingredients to a blender and process until smooth.",
         "utensils" : [],
         "appliances" : [
            {
               "appliance" : "blender"
            }
         ],
         "vessels" : []
      }
   ]
}
```

**Comment by queencitra** [comment]
I can't open `data` as you said. it's not working on my end. I can't say it displays incorrect but it doesn't display enough knowledge on `ingredients` tab.
I saw we only take partial, instead of whole. I prefer to display description of ingredients. e.g. it's much better as`8 ounces, fluid aloe vera drink, mangosteen and mango flavor` and `½ cups mango, frozen` than `225 g mango` and `120 ml mango`.
I got lost when see it. I need to open the link to check it. We scrap it and display it as a simple version.
